Odata-docs/client/annotation-support.md

```diff
@@ -12,14 +12,13 @@ ms.topic: article
 # Client Annotation Support
 **Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
 
-Before ODataLib 6.10.0, OData core lib has supported metadata annotations for metadata element in model and instance annotations for a particular instance in payload. But on client side, there isn’t a good way to get these  annotations. So In ODataLib 6.10.0, we provided several APIs to enable user to get annotations on client side. Basically, OData client follows the rules defined in [OData V4.0 protocol](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html) (see [6.4 Vocabulary Extensibility](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398214)) to get instance annotations or metadata annotations.
+Before ODataLib 6.10.0, OData core lib has supported metadata annotations for metadata element in model and instance annotations for a particular instance in payload. But on client side, there isn’t a good way to get these  annotations. Since ODataLib 6.10.0, we provided several APIs to enable user to get annotations on client side. Basically, OData client follows the rules defined in [OData V4.0 protocol](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html) (see [6.4 Vocabulary Extensibility](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398214)) to get instance annotations or metadata annotations.
 
 ## How to get annotations on client side 
 
-All client CLR types in this tutorial are generated by OData Client Code Generator. Before we dive into this tutorial, you can read “[How to use OData Client Code Generator to generate client-side proxy class](https://devblogs.microsoft.com/odata/tutorial-sample-how-to-use-odata-client-code-generator-to-generate-client-side-proxy-class/)” for generating client side proxy class.
-OData Client provided following APIs for getting annotations in `DataServiceContext` class. 
+All client CLR types in this tutorial are generated by OData Connected Service. Before we dive into this tutorial, checkout [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.
```

Comment:
```suggestion
All client CLR types in this tutorial are generated using OData Connected Service extension. Before we dive into this tutorial, checkout [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.
```

Reply:
Perhaps add article `the` to make it `using the OData Connected Service extension`

Reply:
#Resolved